src/main/java/de/False/BuildersWand/Main.java

```diff
@@ -44,8 +45,13 @@ public void onEnable()
         setupNMS();
         wandManager = new WandManager(this, nms);
         inventoryManager = new InventoryManager(this, nms);
-        update = new Update(this, 51577);
-        update.sendUpdateMessage();
+        try {
+            Class.forName("com.google.gson.JsonParser");
```

Comment:
Well, it looked like 1.8 wasn't really supported by this plugin. Idk (Gson wasn't delivered in 1.8)...
This fixes that problem by disabling the version checker then